textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/dictionarymatcher/DictionaryMatcher.java

```diff
@@ -1,167 +1,222 @@
-
-                    Attribute attribute = schema.getAttributes().get(fieldIndex);
-                    fieldName = attribute.getFieldName();
-
-                    return getSpanTuple();
-
-                } else {
-                    // Increment the fieldIndex and call getNextTuple to search
-                    // in next field
-                    fieldIndex++;
-                    positionIndex = 0;
-                    return getNextTuple();
-                }
-            } else {
-                // If fieldType is not String. Presently only supporting string
-                // type in dictionary
-                fieldIndex++;
-                positionIndex = 0;
-                return getNextTuple();
-            }
-
-        }
-        // Get the next document
-        else if ((dataTuple = operator.getNextTuple()) != null) {
-            fieldIndex = 0;
-            positionIndex = 0;
-            fields = dataTuple.getFields();
-            return getNextTuple();
-
-        }
-        // Get the next dictionary value
-        else if ((dictionaryValue = dictionary.getNextValue()) != null) {
-            // At this point all the documents in the dataStore are scanned
-            // and we need to scan them again for a different dictionary value
-            fieldIndex = 0;
-            positionIndex = 0;
-
-            operator.close();
-            operator.open();
-            dataTuple = operator.getNextTuple();
-            fields = dataTuple.getFields();
-            return getNextTuple();
-        }
-
-        return null;
-    }
-
-    /**
-     * @about Modifies schema, fields and creates a new span tuple
-     */
-    private ITuple getSpanTuple() {
-        List<Attribute> attributesCopy = new ArrayList<>(schema.getAttributes());
-        attributesCopy.add(SchemaConstants.SPAN_FIELD_NAME_ATTRIBUTE);
-        attributesCopy.add(SchemaConstants.SPAN_KEY_ATTRIBUTE);
-        attributesCopy.add(SchemaConstants.SPAN_BEGIN_ATTRIBUTE);
-        attributesCopy.add(SchemaConstants.SPAN_END_ATTRIBUTE);
-
-        List<IField> fieldListDuplicate = new ArrayList<>(fields);
-        fieldListDuplicate.add(new StringField(fieldName));
-        fieldListDuplicate.add(new StringField(dictionaryValue));
-        fieldListDuplicate.add(new IntegerField(spanIndexValue));
-        fieldListDuplicate.add(new IntegerField(positionIndex - 1));
-
-        IField[] fieldsDuplicate = fieldListDuplicate.toArray(new IField[fieldListDuplicate.size()]);
-        return new DataTuple(new Schema(attributesCopy), fieldsDuplicate);
-    }
-
-    /**
-     * @about Closes the operator
-     */
-    @Override
-    public void close() throws DataFlowException {
-        try {
-            operator.close();
-        } catch (Exception e) {
-            e.printStackTrace();
-            throw new DataFlowException(e.getMessage(), e);
-        }
-    }
-}
+
+package edu.uci.ics.textdb.dataflow.dictionarymatcher;

```

Comment:
@chenlica .. I think it is because of the eclipse formatting issues. It shows as if we deleted the file and recreated a new one. Next time, we will be careful before committing such changes.